
# [CVE-2020-29396](https://github.com/odoo/odoo/issues/63712)

## Description

- `A sandboxing issue in Odoo Community 11.0 through 13.0 and Odoo Enterprise 11.0 through 13.0, when running with Python 3.6 or later, allows remote authenticated users to execute arbitrary code, leading to privilege escalation.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 6.5

## Scores

- **Exploitability Score**:
  - 8.0
- **Impact Score**:
  - 6.4
- **Base Severity**:
  - MEDIUM

## Other Information

- **Publish Date**:
  - 2020-12-22 17:15:13
- **Vulnerability Status**:
  - Analyzed

## References

- **security@odoo.com**: https://github.com/odoo/odoo/issues/63712
- **security@odoo.com**: https://www.oracle.com/security-alerts/cpujul2022.html
